--- /Users/frankreyesgarcia/Documents/WORK/PHD/Transformer/output/0abf7148300f40a1da0538ab060552bca4a2f1d8/biapi/src/main/java/xdev/tableexport/export/ReportBuilder.java	2025-12-05 19:52:47
+++ /Users/frankreyesgarcia/Documents/WORK/PHD/Transformer/transformer-agent/./reports/gemini-3-pro-preview/0abf7148300f40a1da0538ab060552bca4a2f1d8/transformed/xdev/tableexport/export/ReportBuilder.java	2025-12-05 19:33:43
@@ -366,7 +366,7 @@
                         return;
                 }
                 
-                textField.getLineBox().getPen().setLineWidth(border.getLineWidth());
+                textField.getLineBox().getPen().setLineWidth(java.lang.Float.valueOf(border.getLineWidth()));
                 textField.getLineBox().getPen().setLineColor(border.getLineColor());
                 textField.getLineBox().getPen().setLineStyle(border.getLineStyle().getLineStyleEnum());
         }
